lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -106,7 +106,10 @@ instead of something meaningful for the client, like this:
 
 To solve these problems, you can configure your reverse proxy to send the missing data (see L</Nginx> and
 L</"Apache/mod_proxy">) and tell your application about it by setting the environment variable C<MOJO_REVERSE_PROXY>.
-For finer control, L</Rewriting> includes examples of how the changes could be implemented manually.
+In more complex situations, usually involving multiple proxies or proxies that live outside your network, it can be
+necessary to tell the application from which ip addresses to expect proxy requests by setting C<MOJO_TRUSTED_PROXIES>
+to a list of comma separated addresses or CIDR networks. For even finer control, L</Rewriting> includes examples of how
+the changes could be implemented manually.
```

Comment:
I think my own misunderstanding of this PR (which I resolved via discussion on IRC) would've benefited from a concrete example here similar to the ones above this paragraph; essentially explicitly pointing out that `X-Forwarded-For: 7.7.7.7, 192.0.2.2, 192.0.2.1` + `MOJO_TRUSTED_PROXIES='127.0/8, 192.0.2.1'` = `192.0.2.2`.

```pod
In more complex situations, usually involving multiple proxies or proxies that live outside your network, it can be
necessary to tell the application from which ip addresses to expect proxy requests by setting C<MOJO_TRUSTED_PROXIES>
to a list of comma separated addresses or CIDR networks.

For example, setting C<MOJO_TRUSTED_PROXIES> to C<127.0/8, 192.0.2.1> will resolve an C<X-Forwarded-For> header value of
C<7.7.7.7, 192.0.2.2, 192.0.2.1> to C<192.0.2.2>.

For even finer control, L</Rewriting> includes examples of how the changes could be implemented manually.
```